### PR TITLE
Improved test_topo 

### PR DESCRIPTION
**Pull request by erik808**
Now it actually checks things that I don't want to break.

### CODE REVIEW

File: src/tests/test_topo.C [hunk 1]
```diff
@@ -163,10 +147,37 @@ TEST(Topo, SpinupContinuation)
...
+    EXPECT_EQ(failed, false);
+}
+
 //------------------------------------------------------------------
 TEST(Topo, TopoContinuation)
 {
 	bool failed = false;
+
+    // create state diff vector
+    Ocean::VectorPtr stateDiff = ocean->getState('C');
```
Comment (Sbte) [inline src/tests/test_topo.C:175]: Why is this called stateDiff?

File: src/tests/test_topo.C [hunk 2]
```diff
@@ -163,10 +147,37 @@ TEST(Topo, SpinupContinuation)
...
+
+    // ocean norm spinup topography
+    double startNorm = Utils::norm(stateDiff);
+    int nMasks;
+    std::vector<double> norms;
 	try
```
Comment (Sbte) [inline src/tests/test_topo.C:181]: Can you fix indentation while you're at it?

File: src/tests/test_topo.C [hunk 3]
```diff
@@ -163,10 +147,37 @@ TEST(Topo, SpinupContinuation)
...
 TEST(Topo, TopoContinuation)
 {
 	bool failed = false;
+
+    // create state diff vector
+    Ocean::VectorPtr stateDiff = ocean->getState('C');
+
+    // ocean norm spinup topography
+    double startNorm = Utils::norm(stateDiff);
+    int nMasks;
```
Comment (Sbte) [inline src/tests/test_topo.C:179]: Please check if this gives an uninitialized value warning on one of the test machines

File: src/topo/Topo.H [hunk 4]
```diff
@@ -56,6 +56,11 @@ Topo(Model model, ParameterList pars)
         b_[k+1] = b_[k] + 1;
     }
 
+    // Initializations
```
Comment (Sbte) [inline src/topo/Topo.H:59]: Maybe the name `initialize` is now a bit misleading?

File: src/tests/test_topo.C (hunk 3 above)
Comment (erik808) [reply src/tests/test_topo.C:175]: fixed in f18d0733903d8a2dd14d29db931cbd2625f1238f

File: src/tests/test_topo.C (hunk 1 above)
Comment (erik808) [reply src/tests/test_topo.C]: fixed in f18d0733903d8a2dd14d29db931cbd2625f1238f

**Pull request merged**